orientdb/src/main/scala/akka/stream/alpakka/orientdb/OrientDBSourceStage.scala

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.orientdb
+
+import akka.stream.{Attributes, Outlet, SourceShape}
+import akka.stream.stage.{GraphStage, GraphStageLogic, OutHandler}
+import com.orientechnologies.orient.`object`.db.OObjectDatabaseTx
+import com.orientechnologies.orient.core.command.OCommandResultListener
+import com.orientechnologies.orient.core.db.ODatabaseRecordThreadLocal
+import com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx
+import com.orientechnologies.orient.core.sql.query.{OSQLNonBlockingQuery, OSQLSynchQuery}
+
+import scala.collection.JavaConverters._
+
+final case class Message[T](oDocument: T)
```

Comment:
I wonder if `Message` as a class name is a bit too generic? `OMessage`, `OData` ?
You may consider adding support for a `passThrough`, as well. E.g. the Elasticseach connector got support for it recently and other flows see the same need.